### PR TITLE
{bio}[GCC/13.2.0] RAxML-NG v1.2.2

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
I'm going to borrow this PR to see if tagbot works as intended (I need a older PR to test with before it was merged).

I think it's going to incorrectly tag this as a "workflow" change, but I need to debug it, sorry for the noise.

**Review by Micket — APPROVED** [review]
lgtm (will just need to check that the build does use the new GMP dep as it's not explicitly told so by any configopts

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/c56cadd245f044f30c6ccc913abbe874 for a full test report.

File: easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-1.2.2-GCC-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+}]
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+    ('GMP', '6.3.0'),
+]
+
```
Comment (Micket) [inline easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-1.2.2-GCC-13.2.0.eb:36]: I think you meant to add
```
configopts = '-DUSE_GMP=ON'
```

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-1.2.2-GCC-13.2.0.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/r/RAxML-NG/RAxML-NG-1.2.2-GCC-13.2.0.eb:40]: Thanks for spotting that. I have also enabled the test, just to be sure.

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Review by Micket — APPROVED** [review]

**Comment by Micket** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**